examples/example_glfw_vulkan/main.cpp

```diff
@@ -496,7 +501,8 @@ int main(int, char**)
             ImGui::Checkbox("Another Window", &show_another_window);
 
             ImGui::SliderFloat("float", &f, 0.0f, 1.0f);            // Edit 1 float using a slider from 0.0f to 1.0f
-            ImGui::ColorEdit3("clear color", (float*)&clear_color); // Edit 3 floats representing a color
+            if (ImGui::ColorEdit3("clear color", (float*)&clear_color)) // Edit 3 floats representing a color
+                memcpy(&wd->ClearValue.color.float32[0], &clear_color, 4 * sizeof(float));
 
```

Comment:
Those changes have been removed in the merged commits, favoring a simpler approach of copying in 1 location below (the removed line).